traced/ykrt/src/mt.rs

```diff
@@ -184,6 +184,8 @@ impl Drop for MT {
 /// The innards of a meta-tracer.
 struct MTInner {
     hot_threshold: AtomicHotThreshold,
+    /// How many threads running user code (i.e. their interpreter, whether directly, or JITed) are
+    /// there at the moment?
```

Comment:
One more bike shed and I will leave you alone I promise.

Let's make this shorter and more affirmative:

> The number of threads running the user's interpreter (directly, or via JITted code).

Reply:
Try .